Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -126,7 +130,7 @@ export function runQTest(args: QTestArguments): Result {
      
     let changeAffectedInputListWrittenFile = undefined;
     let changeAffectedInputListWrittenFileArg = {};
-    if (qCodeCoverageEnumType === "DynamicCodeCov"){
+    if (qCodeCoverageEnumType === "DynamicCodeCov" && File.exists(qInputChangesFilePath)){

```

Comment:
>&& File.exists(qInputChangesFilePath) [](start = 51, length = 37)

why is this necessary? Don't we want the ability to run code coverage without the inputChangesFile being set?

Reply:
I need to know if it is a change based code coverage or a full code coverage. I wanted to use the existence of the inputChanges to distinguish these two cases. Without knowing the full code coverage doesn't work. There is another approach which uses a separated /p: to indicated the change based one. This will make the code coverage feature separated from input changes. But the downside is introducing a extra option.